### PR TITLE
Full documentation restructure

### PR DESCRIPTION
**Pull request by ArkaprabhaChakraborty**
This is for full documentation restructure. Eventually, I'll add some of the missing sections and files and complete the restructuring process. Some boiler plates and unnecessary images or sections might be there. Those will be removed too.

### CODE REVIEW

**Comment by ricekot** [comment]
@ArkaprabhaChakraborty will this restructuring break existing links to our documentation? We should avoid doing that.

I also noticed this PR is adding a blog to the docs website? That should be removed because we already have a blog at https://levo.ai/blog/.

**Review by ricekot — COMMENTED** [review]
All docusaurus tutorial related files should be deleted.

File: blog/2021-08-26-quickstart-with-owasp-zap/static/img/docusaurus-plushie-banner.jpeg
Comment (ArkaprabhaChakraborty) [inline blog/2021-08-26-quickstart-with-owasp-zap/static/img/docusaurus-plushie-banner.jpeg:1]: Remove this

File: static/img/docusaurus-social-card.jpg
Comment (ArkaprabhaChakraborty) [inline static/img/docusaurus-social-card.jpg:1]: Remove

File: static/img/docusaurus.png
Comment (ArkaprabhaChakraborty) [inline static/img/docusaurus.png:1]: remove

File: static/img/undraw_docusaurus_mountain.svg
Comment (ArkaprabhaChakraborty) [inline static/img/undraw_docusaurus_mountain.svg:1]: Remove

File: static/img/undraw_docusaurus_react.svg
Comment (ArkaprabhaChakraborty) [inline static/img/undraw_docusaurus_react.svg:1]: Remove

File: static/img/undraw_docusaurus_tree.svg
Comment (ArkaprabhaChakraborty) [inline static/img/undraw_docusaurus_tree.svg:1]: Remove

File: tutorials/tutorial-extras/img/docsVersionDropdown.png
Comment (ArkaprabhaChakraborty) [inline tutorials/tutorial-extras/img/docsVersionDropdown.png:1]: Remove

File: tutorials/tutorial-extras/img/localeDropdown.png
Comment (ArkaprabhaChakraborty) [inline tutorials/tutorial-extras/img/localeDropdown.png:1]: Remove

**Review by ArkaprabhaChakraborty — COMMENTED** [review]
Stock images from docusaurus to be removed

File: docs/getting-started/first_things_first.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+sidebar_position: 0
+---
+
+## Sign Up
+
+## Dashboard Overview
+
+## Components
+
+Levo has two components: the Sensor and the Satellite.
```
Comment (ArkaprabhaChakraborty) [inline docs/getting-started/first_things_first.mdx:13]: This needs correction. I need some help to find out better wording and back it up with actual facts for this section.

**Pull request closed without merging**